scalafmt-tests/src/test/resources/rewrite/SortModifiersInfix.stat

```diff
@@ -0,0 +1,12 @@
+maxColumn = 200 #to not disturb the output too much
```

Comment:
do we need a separate .stat file for this test?

Reply:
Not really, there were separate files for modifiers already that's why I added another one. I do agree that we can just put it into a single file. I will put it together with `inline`, `transparent` and `open`. What do you think?

Reply:
i like this approach 🙂

Reply:
I put them all together into `SortModifiersScala3.stat`